pandas/core/frame.py

```diff
@@ -368,9 +368,7 @@ def _constructor(self):
 
     @property
     def _constructor_expanddim(self):
-        # TODO: Raise NotImplementedError or change note in extending.rst
-        from pandas.core.panel import Panel
-        return Panel
+        raise NotImplementedError("Not supported for DataFrames!")
```

Comment:
The TODO mentions changing the docs extending.rst but that looks to have already been done